jobserver/admin.py

```diff
@@ -8,6 +9,9 @@
 from .models import Org, Project, User
 
 
+logger = structlog.get_logger(__name__)
```

Comment:
I can't see where this is used.  Is there some magic going on here?

Reply:
Nope, just forgot to remove it 😬